src/plugins/three/DebugTilesPlugin.js

```diff
@@ -83,6 +86,33 @@ export class DebugTilesPlugin {
 
 	}
 
+	get enabled() {
+
+		return this._enabled;
+
+	}
+
+	set enabled( v ) {
+
+		if ( v !== this._enabled ) {
+
+			this._enabled = v;
+
+			if ( this._enabled ) {
+
+				this._initExtremes();
+				this._initializeExistingTiles();
+
+			} else {
+
+				this._removeHelpers();
```

Comment:
Material changes are done based on the `colorMode` in the `_onUpdateAfter` function so we'll also want to make sure we revert the material changes here. This is done in the `dispose` function like so:

```js
// reset all materials
this.colorMode = NONE;
this._onUpdateAfter();
```

Now that I'm looking at it I'm wondering if we should just call `this.dispose` when enabled is toggled to "false" and then call `init` again when it's toggled to "true". Then the plugin can clean up after itself in a consistent way, unhook any events so there's no added overhead. That may be a smaller change.